src/styles/components/_image.scss

```diff
@@ -1,65 +1,64 @@
-.bsds-image
-{
+.bsds-image {
   display: block;
   margin-top: 0;
   object-fit: cover;
   object-position: top center;
 
-  &-1to1
-  {
+  &-1to1 {
     @extend .bsds-image;
+
```

Comment:
same re: space after. what level of control do we have here? it feels weird in 2-line blocks, but feels less weird in longer blocks.